src/_guides/language/language-tour.md

```diff
@@ -3636,11 +3641,6 @@ for advice on how to implement a library package.
 <a id="asynchrony"></a>
 ## Asynchrony support
 
-Dart has several language features
-to support asynchronous programming.
-The most commonly used of these features are
-`async` functions and `await` expressions.
-
 Dart libraries are full of functions that
 return Future or Stream objects.
```

Comment:
Add link to types on first occurrence in this section on asynchrony?

> return [Future][] or [Stream][] objects.

[Future]: tbc
[Stream]: tbc